docs/index.md

```diff
@@ -115,6 +115,8 @@ options for deployment:
 
 * [Spark Homepage](http://spark.apache.org)
 * [Spark Wiki](https://cwiki.apache.org/confluence/display/SPARK)
+* [Spark Community](http://spark.apache.org/community.html) resources, including local meetups
```

Comment:
minor - but could this say "including mailing lists and local meetups"? I think it would be good to still mention the mailing list here, but moving this dropped that text.